app/package.json

```diff
@@ -202,6 +199,7 @@
     "react-switcher": "^1.0.2",
     "react-syntax-highlighter": "^10.1.2",
     "react-toggle-switch": "^3.0.4",
+    "react-tooltip": "^3.10.0",
```

Comment:
Could we have used reactstrap tooltips (https://reactstrap.github.io/components/tooltips/)?

Reply:
We could, you just have to manage the toggling and open state by yourself which can be encapsulated in a shared component. This was an easy lib to use though.

Reply:
👍